oncocnv/v7.0/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:20.04 AS download
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      ca-certificates=20211016~20.04.1 \
+      git=1:2.25.1-1ubuntu3.6 && \
+    rm -rf /var/lib/apt/lists/*
+RUN git clone -b v7.0 --depth 1 https://github.com/BoevaLab/ONCOCNV.git
+
+FROM ubuntu:20.04
+LABEL base_image="ubuntu:20.04" \
+      version="1" \
+      software="ONCOCNV" \
+      software.version="v7.0" \
+      about.summary="ONCOCNV - a package to detect copy number changes in Deep Sequencing data" \
+      about.home="https://github.com/BoevaLab/ONCOCNV" \
+      about.documentation="https://github.com/BoevaLab/ONCOCNV/blob/master/README.md" \
+      about.license="GNU General Public License v3.0" \
+      extra.identifiers.biotools="https://bio.tools/ONCOCNV" \
```

Comment:
Should only set biotools id, not url  ie oncocnv here

Reply:
does that mean `biotools:oncocnv` is the ID?

Reply:
no, id is oncocnv